dockers/docker-sonic-gnmi/base_image_files/monit_gnmi

```diff
@@ -1,5 +1,5 @@
 ###############################################################################
 ## Monit configuration for telemetry container
 ###############################################################################
-check program container_memory_gnmi with path "/usr/bin/memory_checker gnmi 419430400"
+check program container_memory_gnmi with path "/usr/bin/memory_checker gnmi 1073741824"
```

Comment:
Could you only increase limit for smartswitch? For non-smartswitch, this memory limit seems too large and dangerous to the whole box.

Reply:
I have updated

Reply:
Can you use a j2 file and reuse most of the monit config?